catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/SearchFormsApplication.java

```diff
@@ -98,5 +102,31 @@ public void init() {
                 .filter(Objects::nonNull)
                 .collect(Collectors.toList()),
         MAPPER::toJson);
+
+    /** Gets me all query templates that are shared with me * */
+    //TODO: This is no longer required, incorporate client-side filtering using new adjustments to Template Transformer
```

Comment:
Right 👍 just keep in mind that the filtering will be pretty simple: just enough to distinguish between system templates and user created templates. The security plugins should be preventing templates that are not shared with the user from returning to the UI.